### DISCUSSION TITLE
Using OpenAI API

### DISCUSSION THREAD

**Discussion by ZduBart** [microsoft/PubSec-Info-Assistant#480] [Q&A] [answered]
Hello, i was wondering if it's possible to use OpenAI api instead od Azure OpenAI services to run this application? It doesn't seem to support it out of the box but did anyone try doing it?

**Comment by ArpitaisAn0maly** [comment, marked as answer]
IA does not offer OpenAI API support out of the box. OpenAI API uses different call structure than Azure Open AI so it is not straight forward switching. you will have to modify IA at many places. it will be custom code.
